### ISSUE TITLE
The quotemark rule changing severity option does not behave as expected

### ISSUE THREAD

**Issue by TuFat-123** [palantir/tslint#3911]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.3.4
- __Running TSLint via__: node.js

#### TypeScript code being linted

```ts
import {Coupon} from "../../../legacy/portal/ts/modules/coupon/Coupon";
```

with `tslint.json` configuration:

```json
    "quotemark": {
      "options": [ "single"],
      "severity": "warning"
    },
```

#### Actual behavior
Marked as error

#### Expected behavior
To be marked as warning

**Comment by MarcoLeko** [comment]
what kind of noob re u?

**Comment by JoshuaKGoldberg** [comment]
@TuFat-123 I get it reported as a warning through the CLI. Can you provide a bit more information on how you're running TSLint?

```
PS C:\Code\test> tslint -c tslint.json -p tsconfig.json -t stylish
C:/Code/test/index.ts:1:13
WARNING: 1:13  quotemark  " should be '
```

**Comment by JoshuaKGoldberg** [comment]
Closing this issue for housekeeping, but please do post again if you have more info @TuFat-123!
